components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -33,7 +34,10 @@ Rule CloneRule(const Rule& rule, bool reverse_patterns = false) {
       ContentSettingsPattern::FromString("https://firstParty/*")) {
     if (!rule.primary_pattern.MatchesAllHosts()) {
       secondary_pattern = ContentSettingsPattern::FromString(
-          "*://[*.]" + rule.primary_pattern.GetHost() + "/*");
```

Comment:
@bridiver can you say a bit about whats going on here.  Both google.com and googleapis.com should be diff eTLD+1, so I think should be unrelated to dealing with private registries

Reply:
@pes10k pulling in your changes was an accident because I was already on your branch